src/openloco/ui.cpp

```diff
@@ -420,6 +420,10 @@ namespace openloco::ui
             cfg.window_resolution = { width, height };
             config::write_new_config();
         }
+
+        auto options_window = WindowManager::find(WindowType::options);
+        if (options_window != nullptr)
+            options_window->moveToCentre();
```

Comment:
Initially, I was a little apprehensive about this change. However, moving it here does mean that the options window will always be centred — regardless of whether the user resizes the window, or the screen mode changes. So this is actually likely the most consistent solution. Nice!